### PR TITLE
Created Work and About

### PR DESCRIPTION
**Pull request by MarcoDDM**
### Description

This pull request includes the middle section of our mobile website project. The purpose of this section is to provide users with information about our past projects and me. To accomplish this, we have included the "Works" section with cards and the "About Me" section. Please refer to Template 2 for reference, and note that we have chosen to stick to this template for consistency.

In order to make this section look polished and professional, we have utilized both CSS Grid and Flexbox in the layout. 

**List of Additions**

- Added "Works" section
- Added "About Me" section
- Used CSS Grid and Flexbox for the layout
- Utilized "ul" and "li" HTML elements for categorization in "Work" section (HTML, CSS, and JavaScript)
- Utilized "ul" and "li" HTML elements for categorization in "About Me" section (Languages, Frameworks, and Skills)

Thank you for your time and consideration.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -120,3 +120,439 @@
...
+  display: flex;
+  position: absolute;
+  width: 100%;
+  height: 1901px;
+  left: 0;
+  top: 4500px;
+
+  /* Theme 3 / Neutral 1 */
+  background: #1c1a19;
+}
```
Comment (franclobo) [inline style.css:370]: - [x] Please 🙏🏽 add space above the about section to me similar to the Figma template 👇🏽 

| Figm template | Your template |
|---|---|
| image | image |

File: style.css [hunk 2]
```diff
@@ -120,3 +120,439 @@
...
+  list-style-type: none;
+  font-size: 23px;
+  font-family: "Meera Inimai", serif;
+  position: absolute;
+  text-align: center;
+  right: 0;
+  left: 77px;
+  justify-content: center;
+  top: 736px;
+}
```
Comment (franclobo) [inline style.css]: - [x] Kindly add spaces between `li` items on each list to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| image | image |

File: style.css [hunk 3]
```diff
@@ -120,3 +120,439 @@
...
+  list-style-type: none;
+  font-size: 23px;
+  font-family: "Meera Inimai", serif;
+  position: absolute;
+  text-align: center;
+  right: 0;
+  left: 77px;
+  justify-content: center;
+  top: 1000px;
+}
```
Comment (franclobo) [inline style.css]: - [x] Kindly add spaces between `li` items on each list to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| image | image |

File: style.css [hunk 4]
```diff
@@ -120,3 +120,439 @@
...
+  list-style-type: none;
+  font-size: 23px;
+  font-family: "Meera Inimai", serif;
+  position: absolute;
+  text-align: center;
+  right: 0;
+  left: 77px;
+  justify-content: center;
+  top: 1300px;
+}
```
Comment (franclobo) [inline style.css]: - [x] Kindly add spaces between `li` items on each list to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| image |image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @MarcoDDM, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Better practices of HTML & CSS 💯 
- Good using of tags 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please 🙏🏽 complete the instructions on your README file to use your app for professional documentation 👇🏽 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @MarcoDDM  , I'm @MaganaAsiati , your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Kindly is better to use the appropriate font family :+1: it will make you design stick to the Figma design :+1: 
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] Check the comments under the review.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏

**Pull request merged**